mappings/net/minecraft/item/Items.mapping

```diff
@@ -181,6 +181,44 @@ CLASS net/minecraft/class_1802 net/minecraft/item/Items
 	FIELD field_23848 POLISHED_BLACKSTONE Lnet/minecraft/class_1792;
 	FIELD field_23849 POLISHED_BLACKSTONE_SLAB Lnet/minecraft/class_1792;
 	FIELD field_23983 CHAIN Lnet/minecraft/class_1792;
+	FIELD field_27018 COPPER_ORE Lnet/minecraft/class_1792;
+	FIELD field_27019 TINTED_GLASS Lnet/minecraft/class_1792;
+	FIELD field_27020 CALCITE Lnet/minecraft/class_1792;
+	FIELD field_27021 TUFF Lnet/minecraft/class_1792;
+	FIELD field_27024 CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27025 WHITE_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27026 ORANGE_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27027 MAGENTA_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27028 LIGHT_BLUE_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27029 YELLOW_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27030 WEATHERED_CUT_COPPER Lnet/minecraft/class_1792;
+	FIELD field_27031 CUT_COPPER_STAIRS Lnet/minecraft/class_1792;
+	FIELD field_27032 LIGHTLY_WEATHERED_CUT_COPPER_STAIRS Lnet/minecraft/class_1792;
+	FIELD field_27033 SEMI_WEATHERED_CUT_COPPER_STAIRS Lnet/minecraft/class_1792;
+	FIELD field_27052 LIME_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27053 PINK_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27054 GRAY_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27055 LIGHT_GRAY_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27056 CYAN_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27057 PURPLE_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27058 BLUE_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27059 BROWN_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27060 GREEN_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27061 RED_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27062 BLACK_CANDLE Lnet/minecraft/class_1792;
+	FIELD field_27064 AMETHYST_BLOCK Lnet/minecraft/class_1792;
+	FIELD field_27065 BUDDING_AMETHYST Lnet/minecraft/class_1792;
+	FIELD field_27066 SMALL_AMETHYST_BUD Lnet/minecraft/class_1792;
+	FIELD field_27067 MEDIUM_AMETHYST_BUD Lnet/minecraft/class_1792;
+	FIELD field_27068 LARGE_AMETHYST_BUD Lnet/minecraft/class_1792;
+	FIELD field_27069 AMETHYST_CLUSTER Lnet/minecraft/class_1792;
+	FIELD field_27071 COPPER_BLOCK Lnet/minecraft/class_1792;
+	FIELD field_27072 LIGHTLY_WEATHERED_COPPER_BLOCK Lnet/minecraft/class_1792;
+	FIELD field_27073 SEMI_WEATHERED_COPPER_BLOCK Lnet/minecraft/class_1792;
+	FIELD field_27074 WEATHERED_COPPER_BLOCK Lnet/minecraft/class_1792;
+	FIELD field_27075 CUT_COPPER Lnet/minecraft/class_1792;
+	FIELD field_27076 LIGHTLY_WEATHERED_CUT_COPPER Lnet/minecraft/class_1792;
+	FIELD field_27077 SEMI_WEATHERED_CUT_COPPER Lnet/minecraft/class_1792;
```

Comment:
I thought stitch would auto name these?

Reply:
Nothing was auto named. sadly.

Reply:
Humm, this is fine for now. Might be worth looking into why it didnt